src/utils/config.ts

```diff
@@ -22,6 +22,12 @@ const configParsers = {
 
 		return key;
 	},
+	language(key: string) {
```

Comment:
Maybe this should be `locale`. And in the validation, it should say "Must be a valid locale (letters and dashes/underscores)".

Reply:
Changed it! :+1: 

I added a link to the Wikipedia list. I tried to get a link from the ISO organization but it was not possible. But I think the Wiki list is complete and as a reference could be good.